cmd/neofs-cli/modules/root.go

```diff
@@ -123,9 +124,11 @@ func initConfig() {
 			os.Exit(1)
 		}
 
-		// Search config in home directory with name ".main" (without extension).
-		viper.AddConfigPath(home)
-		viper.SetConfigName(".config/neofs-cli")
+		// Search config in `$HOME/.config/neofs-cli/` or `$PWD` with name "config.yaml"
+		viper.AddConfigPath(filepath.Join(home, ".config", "neofs-cli"))
+		viper.AddConfigPath(".")
```

Comment:
I am not sure it is good idea to look for `config.yaml` file in `$PWD` by default. We can accidentally use config of other application that can also have `rpc` of `wallet` section. This won't happen if it is located in separate `.config/neofs-cli` dir. Let's stick only with that.